cli/src/deploy/mod.rs

```diff
@@ -109,14 +109,14 @@ const CREDENTIALS_FILE: &str = ".anycloud/credentials.json";
 pub async fn add_cred() {
   let mut credentials = get_creds().await;
   let clouds = vec!["AWS", "GCP", "Azure"];
-  let selection = Select::new()
+  let selection = Select::with_theme(&ColorfulTheme::default())
```

Comment:
![the colors, duke! The colors!](https://animalnewsthepodcast.files.wordpress.com/2012/05/the_colors_duke.gif)